fs/source/source.go

```diff
@@ -111,36 +111,38 @@ func FromDefaultLabels(hosts RegistryHosts) GetSources {
 			return nil, err
 		}
 
+		var targetSize int64
```

Comment:
nit: the following line will initialize to 0 so this is redundant.

Reply:
The following line creates the string version, not the int.

Reply:
Whoops. Reading too quickly.